README.md

```diff
@@ -435,6 +445,8 @@ For direct execution, you can make multiple calls and combine the resulting PDFs
 
 ### Using middleware
 
+To use this functionality you need add [combine_pdf](https://rubygems.org/gems/combine_pdf) gem in your app.
```

Comment:
```suggestion
Note, to use this functionality you need to add the [combine_pdf](https://rubygems.org/gems/combine_pdf) gem to your app.
```

Also, I think this paragraph should be moved down one (just below the "You can specify relative paths to the cover page contents using..." one).